composer.json

```diff
@@ -52,6 +52,12 @@
 		"mockery/mockery": "^0.9",
 		"kdyby/request-stack": "^0.1.1"
 	},
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "https://github.com/jiripudil/CsobPaymentGateway"
```

Comment:
you have push rights AFAIK, you could have created the branch directly in `Kdyby/CsobPaymentGateway` :)